doc/source/index.rst

```diff
@@ -1,47 +1,60 @@
 Antenna toolkit documentation |version|
 =======================================
 
+**Useful links**:
+:ref:`Installation <installation>` |
+`Source repository <https://github.com/ansys/pyaedt-toolkits-antenna>`_ |
+`Issues <https://github.com/ansys/pyaedt-toolkits-antenna/issues>`_
+
 The Antenna Toolkit is a Python interface for accelerating antenna design 
 using `Ansys Electronics Desktop <https://www.ansys.com/products/electronics>`_
-(AEDT). You can launch this toolkit from the AEDT UI or launch it directly
+(AEDT). You can launch this toolkit from AEDT or launch it directly
 from a Python console.
 
 .. grid:: 2
 
     .. grid-item-card:: Getting started :fa:`person-running`
         :padding: 2 2 2 2
-        :link: getting-started
+        :link: getting_started/index
         :link-type: doc
 
-        Learn more about the Antenna Toolkit and how to install it.
+        Learn more about the Antenna toolkit and how to install it.
 
-    .. grid-item-card:: UI reference :fa:`book-bookmark`
+    .. grid-item-card:: Antenna wizard :fa:`scroll`
```

Comment:
Again, match the name of the wizard in the UI and use it consistently. I believe Wizard would be capitalized--and there likely isn't "Toolkit" in the name, although there are occurrences in this code of Antenna Toolkit Wizard.